light_controlled_intersection_tactical_plugin/src/light_controlled_intersection_tactical_plugin.cpp

```diff
@@ -71,14 +71,15 @@ namespace light_controlled_intersection_tactical_plugin
         }
 
         // get the lanelet that is on the route in case overlapping ones found
-        for (auto llt : current_lanelets)
-        {
-            auto route = wm_->getRoute()->shortestPath();
-            if (std::find(route.begin(), route.end(), llt) != route.end())
-            {
-                current_lanelet = llt;
-                break;
-            }
+        auto llt_on_route_optional = wm_->getFirstLaneletOnShortestPath(current_lanelets);
+
+        if (llt_on_route_optional){
+            current_lanelet = llt_on_route_optional.value();
+        }
+        else{
+            RCLCPP_WARN_STREAM(rclcpp::get_logger("plan_delegator"), "When adjusting maneuver for maneuvers other than lane follow, no previous lanelet found on the shortest path. " <<
```

Comment:
Similar comment to comment on lci_strategic_plugin.cpp, need to change to `rclcpp::get_logger("light_controlled_intersection_tactical_plugin")` and also adjust the text in this log.

Reply:
Fixed the name and description